projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/occ-endpoint.model.migration.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+  OCC_ENDPOINT,
+  BASE_SITES_FOR_CONFIG,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const OCC_ENDPOINT_MIGRATION: MethodPropertyDeprecation[] = [
```

Comment:
`OccEndpoints` is an interface, actually we removed an optional property from it. We an document this change, but I don't think it belongs to `method-and-properteis-deprecations`.